config/config.go

```diff
@@ -0,0 +1,153 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// The MIT License (MIT)
+// Copyright (c) 2018 Tom Kralidis
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to
+// deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+///////////////////////////////////////////////////////////////////////////////
+
+package config
+
+import (
+	"github.com/BurntSushi/toml"
+)
+
+var Configuration Config
+
+func init() {
+	Configuration = Config{
+		Server: Server{
+			MimeType:    "application/json",
+			Encoding:    "utf8",
+			Language:    "en-US",
+			PrettyPrint: false,
+			Limit:       10,
+		},
+		Logging: Logging{
+			Level:   "NONE",
+			Logfile: "",
+		},
+		Metadata: Metadata{
+			Identification: Identification{
+				Title:             "go-wfs",
+				Description:       "go-wfs is a Go server implementation of OGC WFS 3.0",
+				Keywords:          []string{"geospatial", "features", "collections", "access"},
+				KeywordsType:      "theme",
+				Fees:              "None",
+				AccessConstraints: "None",
+			},
+			Provider: Provider{
+				Name: "Organization Name",
+				URL:  "https://github.com/go-spatial/go-wfs",
+			},
+			Contact: Contact{
+				Name:            "Lastname, Firstname",
+				Position:        "Position Title",
+				Address:         "Mailing Address",
+				City:            "City",
+				StateOrProvince: "Adminstrative Area",
+				PostalCode:      "Zip or Postal Code",
+				Country:         "Country",
+				Phone:           "+xx-xxx-xxx-xxxx",
+				Fax:             "+xx-xxx-xxx-xxxx",
+				Email:           "you@example.org",
+				URL:             "http://example.org",
+				Hours:           "Hours of Service",
+				Instructions:    "During hours of service.  Off on weekends.",
+				Role:            "pointOfContact",
+			},
+		},
+		Providers: Providers{},
+	}
+}
+
+// Config provides an object model for configuration.
+type Server struct {
+	Host        string
+	Port        int
+	Url         string
```

Comment:
I think this should be named better, a url includes a protocol, path, & potentially query params, none of which we want in this string.

Reply:
updating to `Address` (i.e. `Server.Address`)

Reply:
Change Host/Port to BindHost/BindPort to make distinction from Address more clear?